GSoC-Ideas.md

```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
 * _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
 * _Recommended:_ Experience with ElasticSearch and Django would be convenient, but can be learned during the project.
 * _Mentors:_ @dlumbrer, @sduenas, @valeriocos
+
+## Idea: Build Workflow Process for CHAOSS Diversity & Inclusion Badging
+
+[ Micro-tasks and place for questions ](https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging/issues/5)
```

Comment:
The issue "Micro-tasks and place for questions" should contain a series of small tasks for a potential student to complete to demonstrate required skills and willingness to put in the effort. This is also a great reason to start communicating with the CHAOSS members.

Right now, the issue only duplicates the content here, which is not the goal.

Reply:
Sorry about that! The issue is not yet complete. I will work on that now.

Reply:
I would like to merge this PR. Could you please continue to define microtasks? The goal is to have some deliverable that we expect from students to judge their readiness to dive into the mentorship.